### PR TITLE
[Feature] Data Indicators

### PR DESCRIPTION
**Pull request by gertie-sheshe**
## Description

Add Data Indicators Section.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] New feature (non-breaking change which adds functionality)

## Screenshots
Screenshot 2021-08-10 at 22 31 54
Screenshot 2021-08-10 at 22 32 06
Screenshot 2021-08-10 at 22 32 20

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

**Comment by kilemensi** [comment]
QQs @gertie-sheshe 

1. Do we know what is supposed to happen when the icons are clicked in mobile (and desktop? I don't see anything in the XD),
2. Have we tried creating SVG icons from the supplied SVG files? I'm thinking that way we'll only need to switch style instead of images on hover/click.
3. There are merge conflicts on the PR.

**Comment by KhadijaMahanga** [comment]
@gertie-sheshe can we merge with master and resolve conflicts

File: package.json [hunk 1]
```diff
@@ -54,6 +54,7 @@
     "@storybook/addon-essentials": "^6.3.4",
     "@storybook/addon-links": "^6.3.4",
     "@storybook/react": "^6.3.4",
+    "@svgr/webpack": "^5.5.0",
```
Comment (KhadijaMahanga) [inline package.json:57]: dev dependencies

File: src/components/DataIndicators/Mobile.js [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+import React from "react";
+
+import useStyles from "./useStyles";
+
+const MobileScreen = ({ title, items, ...props }) => {
+  const classes = useStyles(props);
+
+  return (
+    <Grid className={classes.smContainer}>
+      <RichTypography className={`${classes.sectionTitle} ${classes.smTitle}`}>
```
Comment (KhadijaMahanga) [inline src/components/DataIndicators/Mobile.js:14]: we can use `clsx`

File: src/components/DataIndicators/Mobile.js [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        {title}
+      </RichTypography>
+      <div>
+        {items?.map((item) => (
+          <Grid container>
+            <div
```
Comment (KhadijaMahanga) [inline src/components/DataIndicators/Mobile.js:20]: `key`

File: src/components/DataIndicators/Tablet.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+
+  return (
+    <Grid container>
+      <Grid item md={7}>
+        <RichTypography className={classes.sectionTitle}>
+          {title}
+        </RichTypography>
+        <Grid className={classes.mdItems}>
+          {items?.map((item) => (
+            <Icon
```
Comment (KhadijaMahanga) [inline src/components/DataIndicators/Tablet.js:35]: `key`

File: src/components/DataIndicators/index.js [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+};
+
+DataIndicators.propTypes = {
+  title: PropTypes.string,
+  items: PropTypes.arrayOf(
+    PropTypes.shape({
+      image: PropTypes.string,
+      title: PropTypes.string,
+      description: PropTypes.string,
+    })
```
Comment (KhadijaMahanga) [inline src/components/DataIndicators/index.js:40]:
```suggestion
    PropTypes.shape({ })
```

File: src/components/DataIndicators/index.js [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+import useStyles from "./useStyles";
+
+import Section from "@/pesayetu/components/Section";
+
+const DataIndicators = ({ title, items, ...props }) => {
+  const classes = useStyles(props);
+
+  return (
+    <div className={classes.root}>
+      <Section>
```
Comment (KhadijaMahanga) [inline src/components/DataIndicators/index.js:17]: thiis component is not limited with our defined section widths
Screenshot from 2021-08-11 16-23-12

**Review by KhadijaMahanga — COMMENTED** [review]
Background Image is missing

File: src/components/DataIndicators/index.js (hunk 5 above)
Comment (gertie-sheshe) [reply src/components/DataIndicators/index.js:40]: Don't we know the object items in this scenario? @KhadijaMahanga

File: src/components/DataIndicators/index.js (hunk 6 above)
Comment (gertie-sheshe) [reply src/components/DataIndicators/index.js:17]: Got it

**Review by nerdlet — COMMENTED** [review]
@gertie-sheshe Looks Good!
Quick review from me:
1. There is a need for a smooth transition on the blue box on the right when you click the icon.
 Implementation: 
https://user-images.githubusercontent.com/3693563/129163314-a8889d09-08db-41a3-938d-2c46cd9c8b50.mov
Designs : https://xd.adobe.com/view/b6a179ba-e25e-489c-b1ed-8a8180ec3563-e3f2/
2. Background image is missing 
Screenshot 2021-08-12 at 11 23 10

3. Spacing: Spacing between icons is a tab bit bigger compared to the designs

File: src/components/DataIndicators/Tablet.js (hunk 4 above)
Comment (gertie-sheshe) [reply src/components/DataIndicators/Tablet.js:35]: Fixed

**Review by KhadijaMahanga — APPROVED** [review]
🚀

File: src/components/DataIndicators/useStyles.js [hunk 7]
```diff
@@ -0,0 +1,127 @@
...
+  align: {
+    textAlign: "center",
+  },
+  iconContainer: {
+    width: "100%",
+    textAlign: "center",
+    marginBottom: typography.pxToRem(16),
+    [breakpoints.up("lg")]: {
+      width: "auto",
+      marginLeft: typography.pxToRem(60),
```
Comment (KobbyMmo) [inline src/components/DataIndicators/useStyles.js:57]:
```suggestion
      marginLeft: typography.pxToRem(30),
      marginRight:typography.pxToRem(30),
```

**Review by KobbyMmo — APPROVED** [review]
I think the margin-left here offsets the grid to the right. I think we should distribute the margins to both left and right to keep it centered.
image

**Review by BrendaBucha — APPROVED** [review]
@KhadijaMahanga 🚀 🚀

**Pull request merged**